Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -216,8 +222,8 @@ function HasElements
 #
 function IsPrBuild
 {    
-    $sourceBranch = $env:Build_SourceBranch
-    $scProvider = $env:Build_Repository_Provider
+    $sourceBranch = GetTaskContextVariable "Build.SourceBranch"
+    $scProvider = GetTaskContextVariable "Build.Repository.Provider" 
```

Comment:
Yeah, but the code looks cleaner this way. There's no penalty in reading a var, it's very fast.